### PR TITLE
4041 team adding message inside message replies

### PR DESCRIPTION
**Pull request by anishregmee**
solved issue inside teams and ourNews. @lmmrssa @paulbert

### CODE REVIEW

File: src/app/news/news-list.component.ts [hunk 1]
```diff
@@ -27,14 +27,18 @@ export class NewsListComponent implements OnChanges {
   replyObject: any = {};
   replyViewing: any = { _id: 'root' };
   deleteDialog: any;
+  @Output() showForm = new EventEmitter<any>();
```
Comment (lmmrssa) [inline src/app/news/news-list.component.ts]: Only one output should be enough

File: src/app/news/news-list.component.ts [hunk 2]
```diff
@@ -27,14 +27,18 @@ export class NewsListComponent implements OnChanges {
...
 
   constructor(
     private dialog: MatDialog,
     private dialogsFormService: DialogsFormService,
     private dialogsLoadingService: DialogsLoadingService,
     private newsService: NewsService,
     private planetMessageService: PlanetMessageService
-  ) {}
+  ) {
+    console.log(this.replyViewing);
```
Comment (lmmrssa) [inline src/app/news/news-list.component.ts]: remove unnecessary code

File: src/app/news/news-list.component.ts [hunk 3]
```diff
@@ -47,6 +51,8 @@ export class NewsListComponent implements OnChanges {
   showReplies(news) {
     this.replyViewing = news;
     this.displayedItems = this.replyObject[news._id];
+    this.showForm.emit(this.replyViewing);
+    this.showPost.emit(this.replyViewing);
```
Comment (lmmrssa) [inline src/app/news/news-list.component.ts]: make single event

File: src/app/news/news.component.html [hunk 4]
```diff
@@ -7,14 +7,16 @@
     <mat-toolbar-row class="primary-color font-size-1"></mat-toolbar-row>
   </mat-toolbar>
   <div class="view-container view-full-height">
-    <div *planetAuthorizedRoles="'learner'" class="new-post-form">
-      <mat-form-field class="full-width mat-form-field-type-no-underline">
+    <ng-container *ngIf="showPost">
+      <div *planetAuthorizedRoles="'learner'" class="new-post-form">
+        <mat-form-field class="full-width mat-form-field-type-no-underline">
         <planet-markdown-textbox class="full-width" [(ngModel)]="newMessage" i18n-placeholder placeholder="What would you like to share?"></planet-markdown-textbox>
```
Comment (lmmrssa) [inline src/app/news/news.component.html]: fix indentation

File: src/app/teams/teams-view.component.html [hunk 5]
```diff
@@ -37,8 +37,8 @@ <h3 class="margin-lr-3 ellipsis-title">{{team?.name}}</h3>
         </ng-template>
       </ng-container>
       <ng-container *ngSwitchCase="'news'">
-        <button mat-stroked-button (click)="openAddMessageDialog()">Add message</button>
-        <planet-news-list [items]="news" viewableBy="teams" [viewableId]="team?._id" editSuccessMessage="Message has been updated successfully"></planet-news-list>
+        <button mat-stroked-button (click)="openAddMessageDialog()" *ngIf="showForm">Add message</button>
+        <planet-news-list [items]="news" viewableBy="teams" [viewableId]="team?._id" (showForm)="hidingAddMessage($event)" editSuccessMessage="Message has been updated successfully" (listChange)="abc($event)"></planet-news-list>
```
Comment (lmmrssa) [inline src/app/teams/teams-view.component.html]: why is there listchange event?

**Comment by anishregmee** [comment]
please review this fixed issue. thank in advance @lmmrssa

**Pull request merged**